File: Prodavnica/Prodavnica/Models/Proizvod.cs
using System.ComponentModel.DataAnnotations;

namespace Prodavnica.Models
{
	public class Proizvod
	{
		public int Id { get; set; }

		[Required, MaxLength(30)]
		public string? Naziv { get; set; }

		[Required]
		public int Cena { get; set; }

		[Required]
		public int Kolicina { get; set; }

		public string? Opis { get; set; }

		public byte[]? Slika { get; set; }

		public List<PorudzbinaProizvod>? PorudzbinaProizvods { get; set; }

		public bool Obrisan { get; set; }

		public Korisnik? Korisnik { get; set; }

		public int KorisnikId { get; set; }
	}
}

File: Prodavnica-UI/my-app/src/models/Proizvod.js
export class ProizvodModel {
    constructor(
      Id,
      Email,
      Naziv,
      Cena,
      Kolicina,
      Opis,
      Slika,
      NarudzbinaProivods,
      Obrisan,
      Korisnik,
      KorisnikId
    ) {
      this.Id = Id;
      this.Email = Email;
      this.Naziv = Naziv;
      this.Cena = Cena;
      this.Kolicina = Kolicina;
      this.Opis = Opis;
      this.Slika = Slika;
      this.NarudzbinaProivods = NarudzbinaProivods;
      this.Obrisan = Obrisan;
      this.Korisnik = Korisnik;
      this.KorisnikId = KorisnikId;
    }
  }